### DISCUSSION TITLE
Error: Fetching git information failed

### DISCUSSION THREAD

**Discussion by igorasilveira** [vercel/next.js#15767] [Help]
Hello guys,

Today I **duplicated** a project already deployed to a Vercel account in order to maintain the same structure but create an entirely different project.

Deploying to Vercel from GitHub, gives me the following error: 🅾️

> Error: Fetching git information failed

However, deploying to Netlify had no problems whatsoever. ✅

Any ideas? 🤔

**Comment by c0b41** [comment]
maybe ask instead vercel discussions https://github.com/vercel/vercel/discussions

**Comment by paulogdm** [comment]
Hello,

Our apologies for the inconvenience. We've fixed the issue and you shouldn't observe any further problems.

If you have any questions, let us know.

**Comment by JuanLandtrust** [comment]
I am currently having this same issue on a vercel build all github test checks pass but vercel has error: 

01:39:38.451 | Error: Fetching git information failed
-- | --

**Comment by paulogdm** [comment]
@JuanLandtrust Can you redeploy and let us know?

**Reply by JuanLandtrust** [reply]
Build succesful on redeploy but preview link is still broken

**Reply by paulogdm** [reply]
What do you mean by "broken"?
